src/common/users.ts

```diff
@@ -562,6 +529,23 @@ export enum LogoutReason {
   KratosSessionAlreadyAvailable = 'kratos session already available',
 }
 
+const getAbsoluteDifferenceInDays: typeof differenceInDays = (date1, date2) => {
+  const startOfDay = (date: Date) =>
```

Comment:
There is a function in date-fns `startOfDay` that you should be use for this

Reply:
Oh, nice. Looks like they literally have almost everything date-related.